core/src/main/java/org/elasticsearch/plugins/RemovePluginCommand.java

```diff
@@ -67,7 +68,7 @@ void execute(Terminal terminal, String pluginName, Map<String, String> settings)
         final Path pluginDir = env.pluginsFile().resolve(pluginName);
         if (Files.exists(pluginDir) == false) {
             throw new UserException(
-                    ExitCodes.USAGE,
+                    ExitCodes.IO_ERROR,
```

Comment:
To be clear, I don't think we should change this to an I/O error. I think I'd be okay with a configuration error though.